Formula/rethinkdb.rb

```diff
@@ -38,7 +38,7 @@ def install
     etc.install "packaging/assets/config/default.conf.sample" => "rethinkdb.conf"
   end
 
-  plist_options :manual => "rethinkdb start"
```

Comment:
Changed this based on this comment https://github.com/Homebrew/homebrew-core/pull/915#issuecomment-222121308
`rethinkdb` doesn't have a `start` command, and just running it without arguments will start a local server which is probably what we want here.